tests/fgcmcalTestBase.py

```diff
@@ -401,6 +402,11 @@ def _testFgcmOutputProducts(self, visitDataRefName, ccdDataRefName, filterMappin
         self.assertFloatsAlmostEqual(photoCalMeanCalMags,
                                      photoCalMags, rtol=1e-2)
 
+        # And the photoCal error is just the zeropoint gray error
+        self.assertFloatsAlmostEqual(testCal.getCalibrationErr(),
+                                     (np.log(10.0) / 2.5) * testCal.getCalibrationMean() *
+                                     fgcmZptGrayErr)
```

Comment:
Ditto, and (fyi, but I don’t insist you do it now!):
https://community.lsst.org/t/the-rules-for-breaking-long-python-lines-is-changing/4005
https://developer.lsst.io/python/style.html#wrap-lines-before-binary-operators